lib/js/app/queryCreator/modules/events/actions.ts

```diff
@@ -4,15 +4,27 @@ import {
   FETCH_COLLECTION_SCHEMA,
   FETCH_COLLECTION_SCHEMA_ERROR,
   FETCH_COLLECTION_SCHEMA_SUCCESS,
+  SCHEMA_COMPUTED,
 } from './constants';
 
-import { EventsActions } from './types';
+import { EventsActions, CollectionSchema } from './types';
 
 export const setEventsCollections = (collections: string[]): EventsActions => ({
   type: SET_EVENTS_COLLECTIONS,
   payload: { collections },
 });
 
+export const schemaComputed = (
```

Comment:
what do you think about putting the verb first if it's an action - `computeSchema`

Reply:
Changed to _computeSchemaSuccess_ to keep common convention across codebase.